virttest/utils_netperf.py

```diff
@@ -281,7 +282,7 @@ def start(self, restart=False):
             logging.info("Start netserver with cmd: '%s'" % server_cmd)
             self.session.cmd_output_safe(server_cmd)
 
-        if not self.is_server_running():
+        if not wait.wait_for(self.is_server_running, 5):
```

Comment:
The logic look good to me.
A bit concern, why do not use `utils_misc.wait_for` that in the same repo ?  Both of them can solve this problem :)
From my checking, `avocado.utils.wait.wait_for` can accept  args for `func`, we do not need it here.

Reply:
Yes, either of them could work, but consider about that `utils_misc` is not a well-defined module and we may restructure it in the future (at least I think so), I prefer use the one in `avocado.utils.wait`, thanks.

Reply:
ACK